src/commctl/client_script.py

```diff
@@ -166,6 +166,32 @@ def _put(self, path, data={}):
             'Unable to create an object at {0}: {1}'.format(
                 path, resp.status_code))
 
+    def _delete(self, path, data={}):
+        """
+        Shorthand for DELETEing.
+
+        :param path: Path to request.
+        :type path: str
+        :param data: Optional dictionary to jsonify and DELETE.
+        :type data: dict
+        :return: None on success, requests.Response on failure.
+        :rtype: None or requests.Response
+        """
+        resp = self._con.delete(path, data=json.dumps(data))
+        # XXX: Sometimes the server returns 410 for a successful
```

Comment:
Should we add this as a bug to fix on the server side? It seems like maybe it should always be ```200 OK``` (https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.7)